### PR TITLE
DEV-1310--Client-Feedback-Ability-to-Create-a-Task-Not-Assigned-to-a-Sensor-Location

### PR DESCRIPTION
**Pull request by jotawarsd**
https://microshare.atlassian.net/browse/DEV-1310
Documentation to create custom incidents not related to any sensor or sensor location using share api in postman

### CODE REVIEW

**Review by JulienBardin — CHANGES_REQUESTED** [review]
1) we said we should change the screenshots to not have staff events but eitheir some leak or "customs" events.

2) I think the examples of json events to produce should show the capability of adding additionnal custom events / maybe even custom "alert" but not sure about the custom "solution", I would keep for now the solution has to be either alert (rodent type of solution) or clean (bathroom type of solution)

3) We also discussed about the location tags in the screenshots to take something more random/custom as these one are a copy of customer tags I believe.

4) make the link with the other doc page and explain that the bundler has to be configured. 

5)  put links to understand the alerts/incident process at the top

**Review by JulienBardin — CHANGES_REQUESTED** [review]
we are missing the information about the recType and how to use a custom recType if needed - requiring a custom bundler.

**Review by JulienBardin — CHANGES_REQUESTED** [review]

File: docs/2/technical/microshare-platform-advanced/creating-custom-incidents.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    "usecase": "Usecase" 
+  }, 
+  "solution": "clean" 
+} 
+``` 
+The above example demonstrates the creation of a task with a custom alert and event type. This can be used to assign ad-hoc tasks which may or may not be related to sensor alerts.
+
+## 7. RecTypes
+---------------------------------------
+The dafault recType for alerts is io.microshare.event.alert.
```
[the remark this answers is not in the document]
Comment (jotawarsd) [reply docs/2/technical/microshare-platform-advanced/creating-custom-incidents.md:179]: The . after alert was supposed to be a full stop :) 
But I can see how that can be misleading, so I'll remove that :>

File: docs/2/technical/microshare-platform-advanced/creating-custom-incidents.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    "global": [ 
+      "custom global tags"
+    ], 
+    "iot": { 
+        "time": "YYYY-MM-DDThh:mm:ss.sssZ"
+    }, 
+    "source": [], 
+    "usecase": "Usecase" 
+  }, 
+  "solution": "clean"
```
[the remark this answers is not in the document]
Comment (jotawarsd) [reply docs/2/technical/microshare-platform-advanced/creating-custom-incidents.md:172]: Okay. So, should I add any explanations pertaining to this?

**Review by JulienBardin — CHANGES_REQUESTED** [review]
just the recTypes in the screenshots are confusing

**Review by JulienBardin — APPROVED** [review]
LGTM - we will update with feedback and testings + we are working on options with Lisa for the "customs events"

**Pull request merged**